src/main.cc

```diff
@@ -361,6 +367,10 @@ int main(int argc, char **argv) {
       case 'h':
         print_help(argv[0]);
         return 0;
+      // case 'S':
+      //   std::cout << "optarg " <<  optarg << std::endl;
+      //   std::cout << presets.getThemePath(std::string(optarg)) << std::endl;
+      //   break;
```

Comment:
Before merge, these comments need to be removed if the code isn't used. Very rarely it makes sense to leave code commented out - there's commit history for viewing old code if you need it. But let's keep that for final cleanup.